src/main/java/com/gluonhq/substrate/util/macos/CodeSigning.java

```diff
@@ -58,24 +69,64 @@
     private static final String ERRLINK = "Please check https://docs.gluonhq.com/ for more information.";
 
     private static final String KEYCHAIN_ERROR_MESSAGE = "errSecInternalComponent";
+    private static final List<String> SANDBOX_KEYS = List.of(
+            "com.apple.security.app-sandbox",
+//            "com.apple.security.cs.allow-jit", "com.apple.security.cs.allow-dyld-environment-variables",
```

Comment:
get rid of these?

Reply:
Makes sense. Those are used by the regular jpackage for the JVM, but native-image shouldn't need them